doc/source/_toc.yml

```diff
@@ -303,6 +303,7 @@ parts:
     chapters:
       - file: ray-references/api
       - file: cluster/usage-stats
+      - file: ray-use-cases/overview
```

Comment:
@pcmoritz I think this placement is perfect here - for now. Once we have more substance, we should definitely move it "up" to the overview section, next to "Ecosystem" or so.